config/initializers/active_model_serializers.rb

```diff
@@ -0,0 +1,166 @@
+module AMS
+  module V09
+    class Serializer < ActiveModel::Serializer
+
+       def serializable_hash(adapter_options = nil,
+                             options = {},
```

Comment:
Layout/SpaceAroundEqualsInParameterDefault: Surrounding space detected in default value assignment.